tests/python/pants_test/test_base.py

```diff
@@ -0,0 +1,573 @@
+# coding=utf-8
+# Copyright 2014 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
Should this have an updated year? Or is it primarily a copy of 2014 code? Looks like the latter.

Reply:
Mm. Good question. Might as well refresh.